src/model/entities/segment.ts

```diff
@@ -241,4 +241,8 @@ export class Segment {
   public getUnsyncedCopy(): Segment {
     return Object.assign(Object.create(Object.getPrototypeOf(this)), this, { id: `${this.id}${UNSYNCED_ID_POSTFIX}`})
   }
+
+  public getIsSegmentUntimed(): boolean {
```

Comment:
Methods returning `boolean` should follow the `isX` or `hasX` naming convention. The `get` here is redundant.